stac_ipyleaflet/core.py

```diff
@@ -6,7 +6,7 @@
 import requests
 
 from ipyleaflet import Map, DrawControl, WidgetControl, TileLayer, Popup
-from stac_discovery.stac_widget import StacDiscoveryWidget
+from .stac_discovery.stac_widget import StacDiscoveryWidget
```

Comment:
Note: I personally prefer using `absolute` path to make sure everything is well installed, but that's not mandatory

```python
from stac_ipyleaflet.stac_discovery.stac_widget import StacDiscoveryWidget
```